src/webextension/widgets/input.css

```diff
@@ -28,8 +28,9 @@
 
 .input {
   padding: 0 8px;
-  font-size: 13px;
+  font-size: 15px;
   color: #0c0c0d;
+  max-width: 242px;
```

Comment:
I see now that the spec intentionally makes the text area wider than the single-line inputs. Setting aside the fact that that looks really broken (I would be unsurprised if we get bugs filed by users about this), the right way to handle this would be to set a `max-width` on all the inputs in `item-fields.js`/`item-fields.css`.